.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -498,7 +498,8 @@ private string NormalizeCharWidth(string text)
         // Parse unit phrase. "万", "億",...
         private string ReplaceUnit(string resultText)
         {
-            foreach (var unit in Config.UnitMap.Keys)
+            var unitMap = Config.UnitMap.OrderBy(o => o.Key.Length).ToDictionary(o => o.Key, p => p.Value);
```

Comment:
This should only happen once in initialization, can you move it out of this method?

Reply:
Same in other platforms.